### DISCUSSION TITLE
How we can call electronjs api in nextjs application

### DISCUSSION THREAD

**Discussion by harshmittal-cyber** [vercel/next.js#42915] [Help]
I am using nextjs application and loading url of localhost in electronjs file and running electron app with nextjs . Now i want to call the electron library function in nextjs at runtime then how should i do it

**Comment by icyJoseph** [comment]
Have you tried with `window.electron`? 

Are you using any of these?

- https://github.com/vercel/next.js/tree/canary/examples/with-electron-typescript 
- https://github.com/vercel/next.js/blob/canary/examples/with-electron
